packages/astro/src/assets/services/service.ts

```diff
@@ -235,12 +262,14 @@ export const baseService: Omit<LocalImageService, 'transform'> = {
 			decoding: attributes.decoding ?? 'async',
 		};
 	},
-	getSrcSet(options) {
-		const srcSet: UnresolvedSrcSetValue[] = [];
-		const { targetWidth } = getTargetDimensions(options);
+	getSrcSet(options): Array<UnresolvedSrcSetValue> {
```

Comment:
I don't personally care all that much, but `UnresolvedSrcSetValue[]` is more common in our codebase